website/src/components/donation/bank-transfer-form.tsx

```
@@ -0,0 +1,140 @@
+		<div>
+			{paid ? (
+				<div className="space-y-4 pb-8">
+					<p>{translations.paymentSuccess}</p>
+					<Link className={linkCn({ variant: 'accent' })} href={`/${lang}/${region}/me/contributions`}>
+						{translations.loginLink}
+					</Link>
+				</div>
+			) : qrBillSvg ? (
+				<>
+					<div className="my-8 flex flex-col items-center justify-center space-y-4">
+						<div className="max-w-full" dangerouslySetInnerHTML={{ __html: qrBillSvg }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)